README.md

```diff
@@ -57,37 +57,16 @@ To get the source code from GitHub, type
     cd ./azure-sdk-for-php
 
 > **Note**
-> 
+>
 > The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [PEAR package manager](http://pear.php.net/manual/en/installation.php).
 
-
 ##Install via Composer
+1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
+It can be installed locally or globally depending on your own needs.
 
-1. Create a file named **composer.json** in the root of your project and add the following code to it:
```

Comment:
I understand your point @yaqiyang, but there is a main difference between the current way and what I'm proposing: the current way is explaining how to create a dependency for this package in an external project, while what I'm proposing is a way to achieve the same as explained at ["Download Source Code"](https://github.com/Azure/azure-sdk-for-php#install-via-composer) section with Composer.
You can read more about this feature at https://getcomposer.org/doc/03-cli.md#create-project, and obviously, you're aimed to execute the command by yourself and check what is the result.
Please, let me know if you have any additional concerns about it.